dashboard/app/serializers/user_serializer.rb

```diff
@@ -72,7 +72,8 @@
 #
 
 class UserSerializer < ActiveModel::Serializer
-  attributes :id, :email, :ops_first_name, :ops_last_name, :district, :ops_school, :ops_gender
+  attributes :id, :email, :ops_first_name, :ops_last_name, :district, :ops_school, :ops_gender,
+             :races, :race_interstitial_shown
```

Comment:
Not sure about this, that looks pretty aligned to me, but I don't mind changing it if anyone cares.